docs/understanding-your-pipeline/data-products/index.md

```diff
@@ -72,6 +72,32 @@ Break down the barriers that exist between data producers and data consumers, by
 
 ![Key elements of a Data Product](images/key-elements-of-a-data-product.png)
 
+**Volume metrics**; data products can identify events ingested in your pipeline that match the configured event specifications. This allows your team to visualize the volume-related metrics for events being tracked to specific application IDs.
```

Comment:
> `volume-related metrics for events being tracked to specific application IDs.` -> `occurence-related metrics for events being tracked with specific Event Specification IDs.`

Not sure about this one. Probably are both Application IDs and Event Specification IDs

Reply:
AFAIK volumes count only the events arriving with the specific Event Specification ID on the `event_specification` context. 
Then the split based on application IDs is shown on the 'application IDs' column.

No strong opinion on the phrasing!

Reply:
You are right, just wanted to keep it simple and concise without introducing too much technical or low level details on an 'Introducing Data Products' page. 
It can be overwhelming for someone who just onboarded, specially if we introduce too much concepts (application IDs, source applications, event-specification IDs, contexts) that can be tricky to understand.